zerver/views/report.py

```diff
@@ -72,7 +79,8 @@ def json_report_error(request, user_profile, message=REQ, stacktrace=REQ,
                       ui_message=REQ(validator=check_bool), user_agent=REQ,
                       href=REQ, log=REQ,
                       more_info=REQ(validator=check_dict([]), default=None)):
-
+    # type: (Any, UserProfile, Any, Any, REQ, Any, Any, Any, REQ) -> HttpResponse
+    # TODO: INCOMPATIBLE_TYPES_WITH_REQ_TYPE_OBJECT_AND_REQ
```

Comment:
Yeah.  Let me look into what the options are for annotating REQ.  For now, I think probably you should focus your efforts on non-view code until we figure out the right way to handle this...